src/core/providers/Cookies.tsx

```diff
@@ -1,21 +1,30 @@
 import clientCookies from 'js-cookie';
+import { DateTime } from 'luxon';
 import React, { createContext, ReactNode, useReducer } from 'react';
 
 import { __CLIENT__ } from 'common/constants/environment';
 import { EventView } from 'events/models/Event';
-import { getEventView } from 'events/utils/eventView';
+import { getEventView, getFrontpageEventView } from 'events/utils/eventView';
 
 /**
  * All available cookies should de declared as optionals here.
  * This lets us have som more control of what cookies are avaiable.
  */
 export interface ICookies {
+  frontpageEventView: EventView;
   eventView: EventView;
+  searchText: string;
+  timeStart: DateTime;
+  timeEnd: DateTime;
 }
 
 const initializeCookies = (inital: { [name: string]: any }): ICookies => ({
   ...inital,
+  frontpageEventView: getFrontpageEventView(inital.eventView),
   eventView: getEventView(inital.eventView),
+  searchText: '',
+  timeStart: DateTime.fromJSDate(new Date(new Date().setDate(new Date().getDate() - 690))),
+  timeEnd: DateTime.fromJSDate(new Date(new Date().setDate(new Date().getDate() + 690))),
```

Comment:
Fixed in upcoming commit

Reply:
The default could probably be to show ongoing events + events 1 month into the future?

Reply:
Sounds good, 
we still need to change the ListView to show all events instead of 7, fetch all events, and filter the events for it to have any effect though